src/sage/topology/simplicial_complex.py

```diff
@@ -5015,7 +5015,8 @@ def facets_for_K3():
         True
     """
     from sage.groups.perm_gps.permgroup import PermutationGroup
-    G = PermutationGroup([[(1,3,8,4,9,16,15,2,14,12,6,7,13,5,10)],
-                         [(1,11,16),(2,10,14),(3,12,13),(4,9,15),(5,7,8)]])
-    return ([tuple([g(i) for i in (1,2,3,8,12)]) for g in G]
-            + [tuple([g(i) for i in (1,2,5,8,14)]) for g in G])
+    G = PermutationGroup([[(1, 3, 8, 4, 9, 16, 15, 2, 14, 12, 6, 7, 13, 5, 10)],
+                          [(1, 11, 16), (2, 10, 14), (3, 12, 13),
```

Comment:
throughout combinat, permutations seem to be written without whitespace; is it helpful to introduce all these spaces?

Reply:
this is one of the pep8 style recommendation. I think we will not enforce it everywhere, but here it could be done.